corehq/apps/userreports/templates/userreports/configurable_reports_home.html

```diff
@@ -5,18 +5,18 @@
 {% requirejs_main 'userreports/js/configurable_reports_home' %}
 
 {% block page_content %}
-  <h1>{% trans "Configurable Reports" %}</h1>
+  <h1>{% trans "Custom Web Reports" %}</h1>
```

Comment:
Same comment about confusion. these are not web only reports, so adding this word makes it less accurate.